doc/reference/reference_lua/box_schema/user_disable.rst

```diff
@@ -0,0 +1,29 @@
+.. _box_schema-user_disable:
+
+===============================================================================
+box.schema.user.disable()
+===============================================================================
+
+.. module:: box.schema
+
+.. function:: box.schema.user.disable(username)
+
+    Revokes ``usage`` and ``session`` permissions from the subject user. Equivalent to the following call:
+
+    ..  code-block:: lua
+        
+        box.schema.user.revoke('{username}','usage,session','universe',nil,{if_not_exists=true})            
+    
+    .. NOTE::
+
+       * ``session`` - (cannot be granted to a role) if is not granted, ``IPROTO_AUTH`` always fails connection to the user, so does ``box.session.su()``
```

Comment:
'Connection to the user' is a bit unclear for me and also it is not clear what is meant about `box.session.su()`. I propose to reword it to something simple:

> allows the binary protocol layer (iproto) to authenticate the user